src/delimiter.js

```diff
@@ -231,28 +231,24 @@ var makeStackedDelim = function(delim, heightTotal, center, options, mode) {
     var repeatHeightTotal = repeatMetrics.height + repeatMetrics.depth;
     var bottomMetrics = getMetrics(bottom, font);
     var bottomHeightTotal = bottomMetrics.height + bottomMetrics.depth;
-    var middleMetrics, middleHeightTotal;
+    var middleMetrics = 0, middleHeightTotal = 0, middleFactor = 1;
```

Comment:
`middleMetrics` isn't a number, it's an object, so it probably shouldn't start out initialized to `0`.  Also, when we initialize and declare variables at the same time, we like to put them on separate `var` lines, so

```
var middleMetrics;
var middleHeightTotal = 0;
...
```